exercises/04_count_rune/04_count_rune.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func CountRune(str string, sbyte rune) int {
```

Comment:
nit: I recommend `r` instead of `sbyte`. Why `sbyte`? What does it mean? `r` at least stands for `rune`.